v2/pkg/controller/mpi_job_controller.go

```diff
@@ -74,21 +74,19 @@ const (
 	sshAuthVolume           = "ssh-auth"
 	sshAuthMountPath        = "/mnt/ssh"
 	sshHomeVolume           = "ssh-home"
-	// TODO(alculquicondor): Make home directory configurable through the API.
```

Comment:
Just a thing to consider: would it make sense to make the whole home directory configurable as opposed to just .ssh directory?

Reply:
I thought about that. But someone creating a container image might want not to use `$HOME/.ssh` but a completely different location. And at the moment I can think of other uses of the home folder from the controller perspective.